src/ReverseString.java

```diff
@@ -20,4 +23,41 @@ public static void reverse(char[] s){
         }
         System.out.println(s);
     }
+
+    public static int reverse(int number){
+        boolean flag = false;
+        if(number < 0) {
+            number = Math.abs(number);
+            flag = true;
+        }
+
+        //System.out.println(number);
+        int reversed = 0;
+        while(number > 0){
+
+            int digit = number % 10;
+            reversed = reversed * 10 + digit;
+            number = number / 10;
+            System.out.println(number);
+            System.out.println(reversed);
+        }
+
+        if(flag)
+            reversed = -reversed;
+        System.out.println(reversed);
+        return reversed;
+    }
+
+    public static int getDigitCount(int number){
```

Comment:
Another way in Java could be, parse this number to string, and get the length of the string.